javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/AgentInitializer.java

```diff
@@ -201,4 +213,28 @@ private static AgentStarter createAgentStarter(
   }
 
   private AgentInitializer() {}
+
+  @SuppressWarnings("SystemOut")
+  static void setSystemProperties(@Nullable String agentArgs) {
+    boolean debug = false;
+    if (agentArgs != null && !agentArgs.isEmpty()) {
+      for (String option : agentArgs.split(";")) {
+        int i = option.indexOf('=');
+        if (i < 0) {
+          System.out.println("Malformed agent argument: " + option);
```

Comment:
elsewhere we use `System.err`, debug logging also goes to `System.err` by default